features/channel/managing_channels/adding_channel_with_a_toggle_for_the_lowest_price.feature

```diff
@@ -0,0 +1,53 @@
+@managing_channels
+Feature: Choosing whether to show the lowest product price or not while creating a channel
+    In order to show the lowest price from 30 days before the product has been discounted only for certain channels
```

Comment:
As the number of days will be configurable, I would be for not using it in scenarios
```suggestion
    In order to show the lowest price before the product has been discounted only for certain channels
```